cross-platform/react-app/src/frontend/CameraSample/PicturesBottomPanel.tsx

```diff
@@ -23,10 +23,26 @@ export interface PicturesBottomPanelProps extends ResizableBottomPanelProps {
   iModel: IModelConnection;
 }
 
+/**
+ * Look up a localized string in the `CameraSampleApp` i18n namespace.
+ * @param key The name of the localized string.
+ * @returns The given localized string.
+ */
 function i18n(key: string) {
   return CameraSampleAppGetLocalizedString("PicturesBottomPanel", key);
 }
 
+/**
+ * Create a memoized localized string for the given key.
+ *
+ * __Note__: This just uses {@link React.useMemo} on the result from {@link i18n}.
+ * @param key The i18n key for the label.
+ * @returns A memoized localized string.
+ */
+function useLabel(key: string) {
```

Comment:
I feel a better name would be: `useLocalizedString`

Reply:
I kind of liked the succinctness of useLabel. However, useLocalizedString is more descriptive, so I made the change.

Reply:
This instance of `useLabel` hasn't been renamed.

Reply:
Thanks. I renamed both functions, but VS Code auto-saved all the files when I renamed the other function, but didn't auto-save the one file when I renamed this function (and I didn't notice).